src/Shell/I18nShell.php

```diff
@@ -64,6 +68,51 @@ public function main()
     }
 
     /**
+     * Inits PO file from POT file.
+     *
+     * @param string|null $language Language code to use.
+     * @return void|int
+     */
+    public function init($language = null)
+    {
+        if (!$language) {
+            $language = strtolower($this->in('What language? Please use the two-letter ISO code, e.g. `en`.'));
+        }
+        if (strlen($language) !== 2) {
+            return $this->error('Must be a two-letter ISO code');
```

Comment:
There is no new feature to add. From what I see the `$language` is just used as folder name, so all you have to do is remove this limitation.